_bibliography/pint.bib

```diff
@@ -1881,6 +1874,23 @@ @misc{Ariel2015
 }
 
 @article{ArteagaEtAl2015,
+	abstract = { In view of the rapid rise of the number of cores in modern
+supercomputers, time-parallel methods that introduce concurrency along
+the temporal axis are becoming increasingly popular.  For the solution
+of time-dependent partial differential equations, these methods can
+add another direction for concurrency on top of spatial
+parallelization.  The paper presents an implementation of the
+time-parallel Parareal method in a C++ domain specific language for
+stencil computations (STELLA).  STELLA provides both an OpenMP and a
```

Comment:
you might need to enclose the abstract in double `{{` to make 'STELLA' appearing in capital letters after BibTeX processing (not applicable on website).